.travis.yml

```diff
@@ -39,12 +39,12 @@ before_script:
   # fileDescriptor0 depending on which version of Go was used
   # to build protoc-gen-go.
   - git checkout ssf/sample.pb.go
-  - dep ensure -v
+  - dep check
 
   # We need to use gofmt built with 1.9+
   # on *all* builds
   # because some vendored files use type alias syntax
-  - /usr/bin/gofmt -w .
```

Comment:
Should update this comment since now we're explicitly _not_ gofmting vendored things (and the other)

Reply:
whoops, thought i got that